github-connector/internal/registration/sendregisterrequest.go

```diff
@@ -1,4 +1,4 @@
-package registerservice
```

Comment:
I would suggest to rename files in the application_registry package. I don't have good names yet. As someone said" "There are only two hard things in Computer Science: cache invalidation and naming things."